packages/components/src/hooks/use-onlongpress.ts

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 
 export const useLongPress = (
-    callback = () => {
+    callback: () => void = () => {
         /** empty function */
     },
     ms = 300
 ) => {
     const [startLongPress, setStartLongPress] = React.useState(false);
 
-    const preventDefaults = e => {
+    const preventDefaults = (e: Event) => {
         e.preventDefault();
         e.stopPropagation();
     };
 
     React.useEffect(() => {
-        let timer;
+        let timer: ReturnType<typeof setTimeout> | undefined;
```

Comment:
maybe it better just to use number for types, instead of ReturnType<typeof setTimeout> | undefined

Reply:
Tried that. But I get the below error
<img width="525" alt="Screen Shot 2022-10-12 at 8 06 06 AM" src="https://user-images.githubusercontent.com/98398322/195247563-2ff58000-cee4-45b0-b89e-2cfab4882612.png">